pages/settings/profile.tsx

```diff
@@ -128,25 +125,12 @@ export default function Settings(props: Props) {
 
   useEffect(() => {
     setSelectedTheme(
-      props.user.theme ? themeOptions.find((theme) => theme.value === props.user.theme) : null
+      props.user.theme ? themeOptions.find((theme) => theme.value === props.user.theme) : undefined
     );
     setSelectedWeekStartDay({ value: props.user.weekStart, label: props.user.weekStart });
     setSelectedLanguage({ value: props.localeProp, label: props.localeLabels[props.localeProp] });
   }, []);
 
-  const handleAvatarChange = (newAvatar) => {
-    avatarRef.current.value = newAvatar;
-    const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
-      window.HTMLInputElement.prototype,
-      "value"
-    ).set;
-    nativeInputValueSetter.call(avatarRef.current, newAvatar);
-    const ev2 = new Event("input", { bubbles: true });
-    avatarRef.current.dispatchEvent(ev2);
-    updateProfileHandler(ev2);
-    setImageSrc(newAvatar);
-  };
-
```

Comment:
Moved inline to [take advantage of type inferences](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/forms_and_events).